File: ListView/app/src/main/java/com/overz/listview/MainActivity.java
package com.overz.listview;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

import com.overz.listview.arrayAdapter.ArrayAdapterActivity;
import com.overz.listview.baseAdapter.BaseAdapterActivity;
import com.overz.listview.simpleAdapter.SimpleAdapterActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button arrayAdapterBtn = findViewById(R.id.array_adapter_btn);
        arrayAdapterBtn.setOnClickListener(view -> {
            Intent intent = new Intent();
            intent.setClass(MainActivity.this, ArrayAdapterActivity.class);
            startActivity(intent);
            Toast.makeText(MainActivity.this,
                    "跳转ArrayAdapterActivity", Toast.LENGTH_SHORT).show();
        });
        Button simpleAdapterBtn = findViewById(R.id.simple_adapter_btn);
        simpleAdapterBtn.setOnClickListener(view ->{
            Intent intent = new Intent();
            intent.setClass(MainActivity.this, SimpleAdapterActivity.class);
            startActivity(intent);
            Toast.makeText(MainActivity.this,
                    "跳转SimpleAdapterActivity", Toast.LENGTH_SHORT).show();
        });
        Button baseAdapterBtn = findViewById(R.id.base_adapter_btn);
        baseAdapterBtn.setOnClickListener(view ->{
            Intent intent = new Intent();
            intent.setClass(MainActivity.this, BaseAdapterActivity.class);
            startActivity(intent);
            Toast.makeText(MainActivity.this,
                    "跳转BaseAdapterActivity", Toast.LENGTH_SHORT).show();
        });


    }
}

File: listview_recyclerview/app/src/main/java/com/overz/list_view/MainActivity.kt
package com.overz.list_view

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.overz.list_view.arrayAdapter.ArrayAdapterActivity
import com.overz.list_view.baseAdapter.BaseAdapterActivity
import com.overz.list_view.list_view_test.ListViewActivity
import com.overz.list_view.list_view_test2.ListView2Activity
import com.overz.list_view.simpleAdapter.SimpleAdapterActivity
import com.overz.list_view_recycler_view.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        array_adapter_btn.setOnClickListener {
            val intent = Intent(this,ArrayAdapterActivity::class.java)
            startActivity(intent)
        }
        simple_adapter_btn.setOnClickListener {
            val intent = Intent(this, SimpleAdapterActivity::class.java)
            startActivity(intent)
        }
        base_adapter_btn.setOnClickListener {
            val intent = Intent(this, BaseAdapterActivity::class.java)
            startActivity(intent)
        }
        list_view1_btn.setOnClickListener {
            val intent = Intent(this, ListViewActivity::class.java)
            startActivity(intent)
        }
        list_view2_btn.setOnClickListener {
            val intent = Intent(this, ListView2Activity::class.java)
            startActivity(intent)
        }

    }

}